src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/EvaluationContext.cs

```diff
@@ -183,6 +187,7 @@ internal sealed class EvaluationContext : EvaluationContextBase
             var localSignatureHandle = (localSignatureToken != 0) ? (StandaloneSignatureHandle)MetadataTokens.Handle(localSignatureToken) : default(StandaloneSignatureHandle);
 
             var currentFrame = compilation.GetMethod(moduleVersionId, methodHandle);
+            var currentSourceMethod = compilation.GetSourceMethod(moduleVersionId, methodToken);
```

Comment:
Minor point: for consistency across `GetMethod` and `GetSourceMethod`, consider having `GetSourceMethod` take a `methodHandle` rather than `methodToken`.